dataplane/windows/ipsets/ipsets.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
+	"github.com/projectcalico/felix/ipsets"
```

Comment:
I would avoid importing `felix/ipsets` package because this package is the windows equivalent and should be at the same level of `felix/ipsets`. We could add linux specific dependencies into `felix/ipsets` later and it will break Windows build.

Reply:
I suppose we need to move the metadata constants out of that package then, although personally I'd recommend doing that at the time a breaking change is added rather than forcing an additional package now.